models/settings.go

```diff
@@ -58,6 +58,11 @@ type Settings struct {
 		// Security Threat Tool enabled
 		STTEnabled bool `json:"stt_enabled"`
 	} `json:"sass"`
```

Comment:
OMG, we have a typo there 🤦

Reply:
And we can't fix it

Reply:
Well, we can, by migrating data, but definitely not now